### DISCUSSION TITLE
How do you use Bach?

### DISCUSSION THREAD

**Discussion by grubeninspekteur** [sormuras/bach#253] [Q&A]
I'm all game when it comes to a module-first Java build tool, but how do you install and use Bach? The Readme doesn't tell, and the main branch seems void of implementations (?)

I understood as much that you are supposed to copy the binary into .bach, but where to get it from?

**Comment by sormuras** [comment]
Hi Tim, this project is still under heavy reconstruction <_here be an animated GIF from the 90'ies_> and therefore I cannot answer this very good question today nor tomorrow - but later this year.

There goes today's code typing: 🚧 http://textfiles.com/underconstruction/ 🚧

**Reply by grubeninspekteur** [reply]
Thanks, looking forward to it!
